Dockerfile

```diff
@@ -9,6 +9,10 @@ RUN apt-get update && \
   apt-get install -y unzip && \
   apt-get install -y default-jre
 
+# install unoconv
+RUN apt-get install -y unoconv
+ENV PYTHONPATH=/usr/lib/python3/dist-packages
```

Comment:
That may not be the best way to do it. But otherwise `/usr/bin/unoconv` (with `#!/usr/bin/env python3`) doesn't find the `uno` module in `dist-packages` (`/usr/lib/python3/dist-packages`). Any alternative suggestion?

Reply:
Not sure I do without testing it myself. If this works and has no detrimental effects to the containers other uses then its probably fine for now 🤷‍♂️ .

Reply:
Okay, fair enough. I understand `dist-packages` comes from modules installed via the package manager (apt). But I didn't quite understand how one is meant to include `dist-packages`. It's doesn't seem to be a common use case for apps anyway given they install from _pypi_. I'll leave it then.